addon/components/basic-dropdown.js

```diff
@@ -225,5 +236,13 @@ export default Component.extend({
       registerAPI(newState);
     }
     return newState;
+  },
+
+  _getDestinationId() {
+    if (testing) {
+      return 'ember-testing';
```

Comment:
While testing you would expect the dropdown to work the same as you would in production. And the client app can make sure that the element they want the dropdown appended to is available in the test environment. But I understand you probably don't want to render the dropdown on the `body` during testing. I'd recommend that *if* `destination` is provided, it be used regardless of whether the app is in testing or not. If `destination` is not provided, then change it to `ember-testing`.

However, if you think that it is more important to make testing simpler, it might be better to just set `renderInPlace` true while testing.